crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/attribute.py

```diff
@@ -27,3 +27,63 @@
 
 
 aaaaaaaaaaaaaaaaaaaaa.lllllllllllllllllllllllllllloooooooooong.chaaaaaaaaaaaaaaaaaaaaaaiiiiiiiiiiiiiiiiiiiiiiinnnnnnnn.ooooooooooooooooooooooooofffffffff.aaaaaaaaaattr
+
+
+# Test that we add parentheses around the outermost attribute access in an attribute
+# chain if and only if we need them, that is if there are own line comments inside
+# the chain.
+x1 = (
+    a1
+    .a2
+    # a
+    .  # b
+    # c
+    a3
+    .a4
+)
+x20 = (
+    a
+    .b
+)
+x21 = (
+    a
+    # trailing name own line
+    .b
+)
```

Comment:
Can you add one example where the outer most attribute chain has leading and trailing comments too?